python/py-soxr/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-soxr
+version             0.3.7
+revision            0
+maintainers         nomaintainer
+categories-append   audio
+platforms           {darwin any}
+supported_archs     noarch
+
+description         High quality, one-dimensional sample-rate conversion library for Python
+long_description    {*}${description}
+license             LGPL-2.1+
+homepage            https://github.com/dofuuz/python-soxr
+
+distname            ${python.rootname}-${version}
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+checksums           rmd160  f2441bee743b8c598e49c19a87057ec9ba473431 \
+                    sha256  436ddff00c6eb2c75b79c19cfdca7527b1e31b5fad738652f044045ba6258593 \
+                    size    296432
+
+python.versions     38 39 310 311
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-cython-devel \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-setuptools_scm
+
+    depends_run-append \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-wheel
+
+    depends_test-append \
+                    port:py${python.version}-pytest
+
+    test.run        yes
```

Comment:
the default `python.test_framework` is set to `pytest` in the `python` PG and adds the `py-pytest` dependency automatically, so please remove these lines above. Additionally it sets the things below that you commented out as well so these can be removed.